commit b02baf74e6c14cc6fcf1e4fa94f1a6734c9cffbe
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Sat Nov 12 19:51:51 2016 +0000

    Cleanup none/tests/nocwd.vgtest tmp dirs.
    
    The none/tests/nocwd creates a really deep temporary directory structure
    that is also inaccessible after the test. This causes issues with some
    build wrappers like koji which fail to cleanup such deep subdirectories.
    So explicitly cleanup the directory after the test.
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1390282
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@16134 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/none/tests/nocwd.vgtest b/none/tests/nocwd.vgtest
index 74e2b4a..f8d4c3b 100644
--- a/none/tests/nocwd.vgtest
+++ b/none/tests/nocwd.vgtest
@@ -1,2 +1,3 @@
 prog: nocwd
 vgopts: -q --trace-children=yes
+cleanup: chmod u+rwx /tmp/wd_test_*; rm -rf /tmp/wd_test_*
